TYPO3.Flow/Classes/TYPO3/Flow/Object/Proxy/ProxyMethod.php

```diff
@@ -154,6 +155,10 @@ public function render()
         $visibility = ($this->visibility === null ? $this->getMethodVisibilityString() : $this->visibility);
 
         $returnType = $this->reflectionService->getMethodDeclaredReturnType($this->fullOriginalClassName, $this->methodName);
+        // TODO: Revise "TypeHandling::isSimpleType" and check if we want to add null.
```

Comment:
That TODO should be moved to the `TypeHandling:: isSimpleType()` method ;)
But IMO we open Pandora's box if we change that.
I'll address that and try to add a test for this with a fix for #900